### DISCUSSION TITLE
so confusing: when to use `Listbox`(select) vs `Menu`(dropdown)

### DISCUSSION THREAD

**Discussion by uptonking** [tailwindlabs/headlessui#458] [Help] [answered]
the more i read the docs, the more confusing it is. 

`select` is almost the same as `dropdown` both in style and behavior. 

and your example shows that they do look the same. 

![select-vs-dropdown](https://user-images.githubusercontent.com/11391549/116269378-c2d8c180-a7b0-11eb-80e4-8f3537a68666.png)

**Comment by adamwathan** [comment, marked as answer]
Hey! They are completely different in terms of behavior — Listbox is for making custom `<select>` controls, which are form controls that have a selected value. Menu is for making dropdown menus, like you see in the menu bar in your operating system.

Menu:

<img width="308" alt="image" src="https://user-images.githubusercontent.com/4323180/116270375-0568b280-a74d-11eb-983a-63f4f0dcd73f.png">

Listbox:

<img width="380" alt="image" src="https://user-images.githubusercontent.com/4323180/116270505-27623500-a74d-11eb-9d75-66723ffff5f7.png">

**Reply by JohnBDonner** [reply]
This (and resulting usage differences) should be more apparent in the documentation 😇
